src/createAnimatedComponent.tsx

```diff
@@ -399,6 +408,34 @@ export default function createAnimatedComponent(
       }
     }
 
+    _unsubscribeWindowDimensions() {
+      this._dimensionsSubscription?.remove();
+    }
+
+    _subscribeWindowDimensions() {
+      this._dimensionsSubscription = Dimensions.addEventListener('change', () =>
+        this._configureLayoutAnimations(this._viewTag)
+      );
```

Comment:
I don't think that each animated component should subscribe to device orientation changes as this may impact performance significantly. Do you think it would be possible to call this method only if the animated view has an exiting animation? I understand the limitation here is that the layout animation needs to be built (configured) on the JS runtime and we cannot modify it after the component is rendered. Maybe it would be possible to somehow inject the current width and height of the screen once the exiting animation starts?